pet-store-vanilla-js/styles.css

```diff
@@ -0,0 +1,300 @@
+@import url('https://fonts.googleapis.com/css2?family=Caveat:wght@700&family=Itim&family=Lobster&family=Raleway:ital,wght@1,500&family=Rubik+Wet+Paint&family=Yanone+Kaffeesatz:wght@300&display=swap');
```

Comment:
Please read the rules in the top-level `README.md` file. 
    "- Don't import libraries from external sources."